src/Microsoft.ML.PipelineInference/TransformInference.cs

```diff
@@ -202,7 +202,7 @@ private bool HasMissingVector()
                     while (cursor.MoveNext())
                     {
                         getter(ref value);
-                        if (value.Count > 0 && value.Values.Any(Single.IsNaN))
+                        if (VBufferUtils.HasNaNs(value))
```

Comment:
what happened with first condition? 
`value.Count > 0`

Reply:
It is no longer necessary. `VBufferUtils.HasNaNs` will return `false` when there are no physical values.

This actually fixed a bug here, because `value.Count` could have been `5`, and `value.Values.Length` could have been `10`, and a `NaN` could have been at the end of the array, and this would have said `value` contained a `NaN`, when really it didn't.